provider/cluster_rosa_classic_resource.go

```diff
@@ -380,6 +380,20 @@ func createClassicClusterObject(ctx context.Context,
 	state *ClusterRosaClassicState, logger logging.Logger, diags diag.Diagnostics) (*cmv1.Cluster, error) {
 
 	builder := cmv1.NewCluster()
+	clusterName := state.Name.Value
+	if len([]rune(clusterName)) > 15 {
```

Comment:
Not sure if ROSA/CS is using runes or just length, worth checking

Reply:
updated to just use length.